recipes/dotty-dict/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "dotty-dict" %}
+{% set version = "1.3.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: eb0035a3629ecd84397a68f1f42f1e94abd1c34577a19cd3eacad331ee7cbaf0
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+    - setuptools_scm
+  run:
+    - python >=3.5
+    - setuptools_scm
+
+test:
+  imports:
+    - dotty_dict
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/dotty-dict/
+  summary: Dictionary wrapper for quick access to deeply nested keys.
+  license: MIT
+  license_file: LICENSE
```

Comment:
It might be helpful to add:

```
dev_url: https://github.com/pawelzny/dotty_dict
doc_url: https://dotty-dict.readthedocs.io
``` 

in the `about` section.